### ISSUE TITLE
$getTagId('Tag name');[FEATURE]

### ISSUE THREAD

**Issue by mariorl** [frangoteam/FUXA#1081]
**Describe the feature**
When you create several controlers, for example several siemens S71500 devices, and there are the same tags names inside them, a problem is raised with $getTagId('Tag name');, because the device is not an argument for the function. This could be addressed giving different tags names for vars inside every device. I didn't test the scope of this issue in other functions who works with IDs.

**Describe the solution you'd like**
May be giving a "device" argument to that function -> $getTagId('Tag name', 'device');

**Comment by MatthewReed303** [comment]
I ran into this issue also trying to use scripting and tag names. I have remote pump stations with OPC-UA and each device is programmed the same with same tag names, I ended up having to run a separate instance of Fuxa on Docker for each pump and just use different ports. Then I had a master Fuxa that opened the other Fuxa pages in an IFrame, it's rough, but is a work around in the meantime. I wasn't going to change tag names on the PLC end, they need to stay the same. I agree having an option to define the device is defiantly needed.
